media/css/mozorg/about-transparency.less

```diff
@@ -0,0 +1,267 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+.data-header {
+    margin: 0 auto -6px;
+    width: 100%;
+
+    img {
+        margin: 0 2%;
+        width: 96%;
+    }
+}
+
+main {
+    .font-size(@largeFontSize);
+    background-color: #fff;
+    margin: 0;
+    padding: 0;
+}
+
+#wrapper {
+    padding-bottom: 0;
+}
+
+#colophon {
+    margin-top: 0;
+    padding-top: @baseLine * 2;
+}
+
+.section {
+    .clearfix;
+    border-top: 1px solid rgba(0, 0, 0, .05);
+    padding: (@baseLine * 2) 0;
+
+    &:nth-child(even) {
+        background: #fcfcfc;
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        padding: @baseLine 0;
+    }
+}
+
+.content,
+.report .section-title {
+    .border-box;
+    margin: 0 auto;
+    overflow: visible;
+    padding: 0 @gridGutterWidth;
+    width: @widthDesktop;
+
+    @media only screen and (min-width: @breakTablet) and (max-width: @breakDesktop) {
+        width: @widthTablet;
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        width: @widthMobile;
+    }
+
+    @media only screen and (min-width: @breakMobileLandscape) and (max-width: @breakTablet) {
+        width: @widthMobileLandscape;
+    }
+}
+
+.section-extra h3 {
+    .font-size(20px);
+}
+
+.title-bar {
+    background: @mozillaRed;
+    border-bottom: 2px solid #eee;
+    padding: (@baseLine * 2) 0;
+
+    h1 {
+        color: #fff;
+        margin: 0;
+        text-shadow: none;
+
+        span {
+            .font-size(24px);
+            display: block;
+            margin: .5em 0 0;
+        }
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        padding: @baseLine 0;
+    }
+}
+
+.section-title {
+    .font-size(42px);
+    color: @mozillaRed;
+    text-shadow: none;
+
+    @media only screen and (max-width: @breakTablet) {
+        .font-size(24px);
+    }
+}
+
+.accordion .section-title {
+    margin-bottom: .5em;
+}
+
+.accordion-initialized .section-title {
+    margin-bottom: 0;
+}
+
+#report-sections.accordion-initialized [data-accordion-role='tab'] {
+    .border-box;
+    overflow: visible;
+    padding: 0 20px 0 40px;
```

Comment:
eh, I've lost faith in that variable (and our grid system in general; we don't consistently stick to it). The point of variables is that they can vary, but more often than not we just use this one as an alias for `20px` in places where the value probably shouldn't change anyway. If by some chance we decided to globally change our entire site's gutter width to 30px, I think I'd still want this particular element to be `padding: 0 20px 0 40px;`. The `@gridGutterWidth` variable is so entrenched and overused at this point changing its value would be disastrous. I'd like to phase it out with the redesign.